apps/emqx/src/emqx_session.erl

```diff
@@ -211,17 +211,17 @@ init(Opts) ->
     #session{
         id = emqx_guid:gen(),
         clientid = maps:get(clientid, Opts, <<>>),
-        is_persistent = maps:get(is_persistent, Opts, false),
-        max_subscriptions = maps:get(max_subscriptions, Opts, infinity),
+        is_persistent = maps:get(is_persistent, Opts),
```

Comment:
Could we use `maps:merge` instead of these `maps:get`?

Reply:
It‘s a record， not a map.